diff --git a/common/build_version.cpp b/common/build_version.cpp
index c05fc41c8..9d93ec59c 100644
--- a/common/build_version.cpp
+++ b/common/build_version.cpp
@@ -54,7 +54,7 @@ wxString GetBuildVersion()

 wxString GetBuildDate()
 {
-    wxString msg = wxString::Format( wxT( "%s %s" ), wxT( __DATE__ ), wxT( __TIME__ ) );
+    wxString msg = wxString::Format( wxT( "unknown" ) );
     return msg;
 }

